src/components/media-player/ha-media-player-browse.ts

```diff
@@ -1326,6 +1331,10 @@ export class HaMediaPlayerBrowse extends LitElement {
         ha-browse-media-tts {
           direction: var(--direction);
         }
+        ha-icon-button {
+          background-color: var(--primary-color);
+          color: var(--primary-text-color) !important;
+        }
```

Comment:
replace content of `ha-card:hover .play:not(.can_expand)` with this content, then you don't need the `!important`